src/components/AddBook.js

```diff
@@ -1,14 +1,39 @@
-import React from 'react';
-
-const AddBook = () => (
-  <div id="form-container">
-    <h2>ADD NEW BOOK</h2>
-    <form>
-      <input type="text" name="title" placeholder="Title" />
-      <input type="text" name="author" placeholder="Author" />
-      <button type="submit" id="add-book">ADD BOOK</button>
-    </form>
-  </div>
-);
+import React, { useRef } from 'react';
+import PropTypes from 'prop-types';
+
+const AddBook = ({ addNewBook }) => {
+  const titleRef = useRef();
+  const authorRef = useRef();
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    const newBook = {
+      id: 3,
```

Comment:
- [ ] Please make sure every time a new book is added, it is given its own unique id. Currently, you are giving every new book an id of 3. This affects the ability to delete a single book because when I add multiple books and delete one, all the books I added will disappear because they all have an id of 3. One way you can solve this is with [uuid](https://www.npmjs.com/package/react-uuid) that auto generates an id for each book you create 🏄